SD-K9/src/app/components/outdoor-map/outdoor-map.component.spec.ts

```diff
@@ -77,4 +77,23 @@ describe('OutdoorMapComponent', () => {
 
   }));
 
+  it('#reverseGeocode should return correct address from coordinates', async() => {
+    await delay(10000);
+
+    let coords = { lat : 45.497061 , lng : -73.578802 }; //Concordia University's coordinates
+    let address = '1455 Boulevard de Maisonneuve O, Montréal, QC H3G 1M8, Canada';
+    component.reverseGeocode(coords, function(test){
+      expect(address).toEqual(test);
+    });
+  });
+
+  it('#clickToMark should place marker and open infoWindow', async() => {
+    await delay(10000);
+
+    let coords = { lat : 45.497061 , lng : -73.578802 };
+    component.clickToMark(coords);
+    expect(component.infowindow).toBeDefined();
+    expect(component.clickMarker).toBeDefined();
```

Comment:
Should this be split into two tests?

Reply:
I don't think it should because the function creates the marker and the InfoWindow at the same time.